docs/python/tutorial_python.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: docs
+title:  Creating Python apps for the Streaming Analytics service
+description:  Learn how to deploy a Python app in the Streaming Analytics service
+weight: 10
+published: true
+---
+
+You can create an app in your Python environment, and then deploy the app to an instance of the Streaming Analytics service running on IBM Bluemix.
+
+The Streaming Analytics service is built on IBM Streams technology. You do not need a local version of IBM Streams to build apps for the service.
+
+This tutorial shows how to build a simple app with Python. The app runs as a job in your Streaming Analytics instance. 
+
+*Note:* Familiarity with Python is recommended. This tutorial is designed to run in a Python 3.5 environment.
+
+<!-- This set up section will probably be removed when the DSX integration with the Streaming Analytics service is enhanced -->
```

Comment:
Why would this section be removed?

This is not a DSX tutorial so any changes in DSX do not affect this.